aiounifi/models/device.py

```diff
@@ -416,7 +416,7 @@ def index(self) -> int:
     @property
     def has_relay(self) -> bool:
         """Is the outlet controllable."""
-        return self.raw.get("has_relay", False)
+        return self.raw.get("has_relay", False) or "relay_state" in self.raw
```

Comment:
I'm trying to work away from doing simplifications in the first level of api exposure to keep it close to the upstream API. As has_relay already expects has_relay to be an optional value this is not weird, but depending on relay_state in the specific value is weird.

Not sure to make this educational though ;)